recipes/ocr_pre_annotations.ipynb

```diff
@@ -1,12 +1,23 @@
 {
```

Comment:
<p>We will now see how to perform OCR preannotation on our image using Google Vision API.</p><p> </p><p>and the itemized lists should be fixed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
bullet point lists don't show well on reviewbn

looks like this on mkdocs:

![image](https://user-images.githubusercontent.com/15847892/223068060-7eb054b5-421a-4e50-929d-e5a30bdbe474.png)